test/test_utils/__init__.py

```diff
@@ -117,9 +117,15 @@ def get_dockerfile_path_for_image(image_uri):
     device_type = get_processor_from_image_uri(image_uri)
     cuda_version = get_cuda_version_from_tag(image_uri)
 
+    dockerfile_name = f"Dockerfile.{device_type}"
```

Comment:
Implementation is different from [that one.](https://github.com/aws/deep-learning-containers/pull/1278/files#diff-d0a5258102fa0a16b552dde7aa9b728f5d9f324e82c0ec43a07dd7c3e89d6f73R89)
Actually seems a good candidate for utility method somewhere in DockerImage. E.g. _get_dokerfile_name_from_image_tag()_.

Reply:
Added test utils function for this